.github/actions/to-integration-tests/entrypoint.sh

```diff
@@ -16,8 +16,8 @@
 # specific language governing permissions and limitations
 # under the License.
 
-+set -e
-+
+set -e
+
```

Comment:
This is #5952. Can this fix go in a separate PR?

Reply:
yes